mlflow/projects/databricks.py

```diff
@@ -233,12 +233,23 @@ def _before_run_validations(tracking_uri, cluster_spec):
             "tracking URI %s." % tracking_uri)
 
 
+def _get_tracking_uri_for_run():
+    if not tracking.utils.is_tracking_uri_set():
+        eprint("No tracking URI was specified, defaulting to using tracking URI 'databricks' for "
+               "Databricks project execution.")
```

Comment:
No need to print this IMO since this will be the common thing people want to do.